pages/data-providers/[data-provider-id]/dois.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import dayjs from 'dayjs'
+
+import { withGlobalStore } from 'store'
+import Table from 'components/table'
+import { Card, Label } from 'design'
+import DocumentLink from 'components/document-link'
+
+// TODO: Remove once cards are in @oacore/design
+// eslint-disable-next-line
+import styles from './dois.css'
+
+const tableConfig = {
+  columns: [
+    {
+      id: 'oai',
+      order: '',
+      display: 'OAI',
+      sortable: true,
+      expandedSize: 2,
+      className: styles.labelColumn,
+      render: (v, isExpanded) => (
+        <Label color="primary">{isExpanded ? v : v.split(':').pop()}</Label>
+      ),
+    },
+    {
+      id: 'title',
+      display: 'Title',
+      order: '',
+      expandedSize: null,
+      className: styles.titleColumn,
+    },
+    {
+      id: 'authors',
+      display: 'Authors',
+      order: '',
+      expandedSize: null,
+      getter: v => v.authors && v.authors.map(a => a.name).join(' '),
+      className: styles.authorsColumn,
+    },
+    {
+      id: 'publicationDate',
+      display: 'Publication date',
+      className: styles.depositDateColumn,
+      expandedSize: 1,
+      getter: v => dayjs(v.publicationDate),
+      render: v => v.format('DD/MM/YYYY'),
+    },
+    {
+      id: 'publicReleaseDate',
+      display: 'Deposit date',
+      order: 'desc',
+      className: styles.depositDateColumn,
+      expandedSize: 1,
+      getter: v => dayjs(v.publicReleaseDate),
+      render: v => v.format('DD/MM/YYYY'),
+    },
```

Comment:
Should be DOI only. I do not thing we need more columns but maybe @mcancellieri can help us here.

```suggestion
    {
      id: 'doi',
      display: 'DOI',
      className: styles.doiColumn,
      expandedSize: 1,
      getter: v => v.doi,
      render: v => v,
    },
```

Reply:
The response contains these fields:

oai, repo_doi, crossref_doi,title, authors. I decided to show everything but oai and crossref_doi. Let me know if it needs to be changed.

Reply:
I guess we need to show something like `repo_doi || crossref_doi`, ideally `crossref_doi && !repo_doi ? crossref_doi 🆕 : repo_doi`